buyer-signing-service/src/routes/sign/batPayTransfer.js

```diff
@@ -79,17 +75,22 @@ const router = express.Router();
  *         example: '"0x7a9d3a032b8ff274f09714b56ba8e5ed776ec9638ca303069bc3a3267bb22f65"'
  *         required: true
  */
-router.post('/bat-pay/transfer', asyncError(async (req, res) => {
-  const { contracts: { batPay } } = req.app.locals;
-  const { nonce, gasPrice, params } = req.body;
-  const sign = builder(batPay, 'transfer');
-  const { errors, result } = sign(nonce, gasPrice, { ...params, fromId: buyer.getId() });
+router.post(
+  '/bat-pay/transfer',
```

Comment:
should the endpoint change as BatPay method changes ?
```suggestion
  '/bat-pay/register-payment',
```

Reply:
Filename also.